qt/src/hocr/HOCRProofReadWidget.cc

```diff
@@ -109,6 +112,8 @@ class HOCRProofReadWidget::LineEdit : public QLineEdit {
 	void keyPressEvent(QKeyEvent* ev) override {
 		HOCRDocument* document = static_cast<HOCRDocument*>(m_proofReadWidget->documentTree()->model());
 
+		#define KEY_IS_ARROW(a) (a == Qt::Key_Up || a == Qt::Key_Down || a == Qt::Key_Left || a == Qt::Key_Right)
```

Comment:
I fully accept whatever well-deserved mocking, derision, or wrinkled noses are thrown my way, as punishment for _this_ nastiness.

Also, I can take that out, if it's deemed too ugly to live. :disguised_face: